rmw_fastrtps_cpp/QUALITY_DECLARATION.md

```diff
@@ -104,7 +111,8 @@ This includes:
 
 Changes are required to make a best effort to keep or increase coverage before being accepted, but decreases are allowed if properly justified and accepted by reviewers.
 
-Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/src_ros2_rmw_fastrtps_rmw_fastrtps_cpp_src/). This package does not yet meet the 95% coverage guideline.
+This package does not meet the 95% coverage guideline, as error handling behavior is currently not being tested on system resource exhaustion and on `fastrtps` internal failure.
+Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/src_ros2_rmw_fastrtps_rmw_fastrtps_cpp_src/).
```

Comment:
I'd rearrange and reword this.

```suggestion
Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/src_ros2_rmw_fastrtps_rmw_fastrtps_cpp_src/).

This package claims to meet the coverage requirements for the current quality level, even though it doesn't have 95% line coverage.  The justification is that the only uncovered lines have to do with system resource exhaustion and Fast-DDS internal failure.
```

Reply:
(same comment for below)

Reply:
Suggestion accepted. See https://github.com/ros2/rmw_fastrtps/pull/456/commits/f035c81af14b2fde6d8761e3748bd7037a301851.